### ISSUE TITLE
Bug : image Links ex) [![](http://example.com/link.png)](http://example.com)

### ISSUE THREAD

**Issue by 100space** [quantizor/markdown-to-jsx#531]
Bug : image Links ex) [![](http://example.com/link.png)](http://example.com)

I want to create a link using an image, but there is a bug in the render. For example:

```
[![youtubeImg](https://www.gstatic.com/youtube/img/promos/growth/ytp_lp2_logo_phone_landscape_300x44.png)](https://www.youtube.com/)
```

![스크린샷 2023-12-12 오후 2 46 07](https://github.com/quantizor/markdown-to-jsx/assets/112856532/9e717132-88d9-49a0-b3c7-01f3e3b434f4)

![스크린샷 2023-12-12 오후 2 46 23](https://github.com/quantizor/markdown-to-jsx/assets/112856532/0dd7abc3-e452-4b0e-bd35-1c6b30d3307f)

Using the above syntax, I want to make a link with an image. It renders correctly in 'vscode', but when using the library, it gets rendered as:

```
![youtubeImg](https://www.youtube.com/)
```

(The image does not appear)

I want it to be rendered as:

```
<p><a href="https://www.youtube.com"><img src="https://www.gstatic.com/youtube/img/promos/growth/ytp_lp2_logo_phone_landscape_300x44.png" alt="youtubeImg"></a></p>
```

But actually, it gets rendered as:

```
<p><a href="https://www.gstatic.com/youtube/img/promos/growth/ytp_lp2_logo_phone_landscape_300x44.png">![youtubeImg</a>](<a href="https://www.youtube.com">https://www.youtube.com</a>)</p>
```

The image tag is lost...

This issue also occurs in the example you have set up (https://markdown-to-jsx.quantizor.dev/)."

<img width="1998" alt="스크린샷 2023-12-12 오후 2 54 40" src="https://github.com/quantizor/markdown-to-jsx/assets/112856532/e9c4c869-d33e-4192-afb4-660d1b54c8a9">

Please check and give me feedback👍

**Comment by quantizor** [comment]
Yup I see it, will look into it

**Comment by quantizor** [comment]
Appears to have been fixed by #551, thanks @zegl!
